### PR TITLE
V1-design | Essentials page (MVP)

### PR DESCRIPTION
**Pull request by Da-Colon**
## Description
Update to the Create Workflow's first screen

## Screenshots (if applicable)
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "private": true,
   "dependencies": {
     "@chakra-ui/react": "^2.3.4",
-    "@decent-org/fractal-ui": "^0.0.7",
+    "@decent-org/fractal-ui": "^0.0.12",
```
Comment (Da-Colon) [inline package.json:6]: v++++, kinda skips a few levels since this was based differently from the font fixes

File: src/routes/constants/dao.ts [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+    relative: (daoAddress: string) => `/daos/${daoAddress}/activities`,
+    path: 'activities',
+  },
+  treasury: {
+    relative: (daoAddress: string) => `/daos/${daoAddress}/treasury`,
+    path: 'treasury',
+  },
+  proposals: {
+    relative: (daoAddress: string) => `/daos/${daoAddress}/proposals`,
+    path: 'proposals',
```
Comment (Da-Colon) [inline src/routes/constants/dao.ts:20]: So decided this object format for the routes. `relative` being the full pathname that may include params that need to be passed in.

`path` is used for the nested routes as it doesn't need the full path

**Review by Da-Colon — COMMENTED** [review]
no big crazy changes, just a reskin

**Review by herbig — APPROVED** [review]

**Review by christopherdancy — APPROVED** [review]
LGTM

**Pull request merged**